config/config.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"gopkg.in/yaml.v3"
+	"os"
+)
+
+type (
+	Config struct {
+		DBConfig DBConfig `yaml:"db_config"`
+		Server   Server   `yaml:"server"`
+	}
+	Server struct {
+		Host string `yaml:"host"`
+		Port string `yaml:"port"`
+	}
+	DBConfig struct {
+		Migrates string `yaml:"migrates"`
+		Driver   string `yaml:"driver"`
+		Postgres Connector
+	}
+	Connector struct {
+		ConnectionDSN string `yaml:"connection_dsn"`
+	}
+)
+
+func (c Connector) DSN() (string, error) {
```

Comment:
i think you can remove it and just use c.ConnectionDNS

Reply:
migrate package asks for this